src/scanner/get-rule-inclusions.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { BestPractice } from 'scanner/rule-to-links-mappings';
+import { DictionaryStringTo } from 'types/common-types';
+import { HyperlinkDefinition } from 'views/content/content-page';
+import { IRuleConfiguration } from 'scanner/iruleresults';
+
+export interface RuleIncluded {
```

Comment:
Can probably be a little stronger in this type

`export type RuleIncluded = { status: 'included' } | { status: 'excluded', reason: string }`

Reply:
went with in-between `{​​​​​ status: 'included', reason?: string }​​​​​ | {​​​​​ status: 'excluded', reason: string }​​​​​`